diagnostic.md

```diff
@@ -150,7 +173,7 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+div and divOne side by side, div 2 3 4 stacked on top of eachother and div 5 to the left of the screen
```

Comment:
.divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).

In short:
```
2 3      1
4 5
```